presto-kinesis/src/main/java/io/prestosql/plugin/kinesis/KinesisConfig.java

```diff
@@ -299,4 +300,18 @@ public KinesisConfig setIteratorNumber(int iteratorNumber)
         this.iteratorNumber = iteratorNumber;
         return this;
     }
+
+    @Min(0)
+    public int getUpdateInterval()
+    {
+        return updateInterval;
+    }
+
+    @Config("kinesis.update-interval")
+    @ConfigDescription("Kinesis update interval")
```

Comment:
"Kinesis update interval" looks too generic. Could you improve this description? Also, this properties is related to `kinesis.table-description-location`. I would move to around L62.

Reply:
done